nservicebus/upgrades/nhibernate-timeouts-index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: NHibernate Persistence - Resolving incorrect timeout table schema #252
+summary: Instructions on how to resolve incorrect schema that can cause performance issues for affected versions 6 to 7.
+component: NHibernate
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 5
+ - 6
+---
+
+
+## Summary
+
+This guidance explains how to resolve an incorrectly created schema when passing a custom NHibernate configuration to NServiceBus as described in the following issue:
+
+- https://github.com/Particular/NServiceBus.NHibernate/issues/252
+
+This issues causes performance issues if the table contains a large number of rows. Inserts and queries are inefficient due to the incorrect order of columns. This results in unnecessary locking which limits the processing throughput of timeouts.
+
+
+## Compatibility
+
+This issues has been resolved in following patch versions as defined in our  [support policy](support-policy.md):
+
+- 7.1.4 in use by NServiceBus 6.x
+- 7.0.6 (support 7.0.x) in use by NServiceBus 5.x
+- 6.2.8 (support 6.2.x) in use by NServiceBus 5.x
+
+
+
+If you are using any of the supported minor versions (7.1.x, 7.0.x, or 6.2.8) then you should at least update to the latest patch release. If you are using an older version then you should update to a newer minor (in case of 6.1.x or 6.0.x) or major version (any version prior to 6.x).
```

Comment:
In this sentence:

> If you are using any of the supported minor versions (7.1.x, 7.0.x, or 6.2.8)

Do you mean supported? Or supported *affected* versions?